src/Surfnet/StepupMiddleware/GatewayBundle/Entity/SecondFactor.php

```diff
@@ -96,11 +96,16 @@ class SecondFactor
     private $secondFactorIdentifier;
 
     /**
-     * @var string
+     * This boolean indicates if the second factor token was vetted
+     * using one of the vetting types that are considered 'identity-vetted'.
+     * That in turn means if the owner of the second factor token has its
+     * identity vetted (verified) by a RA(A) at the service desk. This trickles
+     * down to the self-vet vetting type. As the token used for self vetting
+     * was RA vetted.
      *
-     * @ORM\Column(length=255)
+     * @ORM\Column(type="boolean", options={"default":"1"})
```

Comment:
Is it expected that Doctrine translates this to `VARCHAR(255) NOT NULL DEFAULT \'1\''`?

Reply:
Ahh, that is a remainder of when this column held the `vetting_type` which was a varchar. I'm going to update this. :+1: